Update allowed enum values for severity

begoldsm opened the pull request:
Updating the allowed enum values to include SevereWarning, Deprecated
and UserWarning, which the service has been able to return but were not
incorporated properly in the client.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

Commit:
Update allowed enum values for severity

Updating the allowed enum values to include SevereWarning, Deprecated
and UserWarning, which the service has been able to return but were not
incorporated properly in the client.

azurecla commented:
Hi __@begoldsm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

matt1883 committed:
Update filesystem.json

Clarifying helptext for ConcurrentAppend, Append, and Create.

begoldsm committed:
Merge pull request #4 from matt1883/patch-2

Clarifying helptext for ConcurrentAppend, Append, and Create.

amarzavery commented on arm-datalake-analytics/job/2016-11-01/swagger/job.json:
```diff
@@ -700,7 +700,10 @@
             "Warning",
             "Error",
-            "Info"
+            "Info",
```
This will be a breaking change for strongly typed languages or languages that support enums.

begoldsm replied:
@amarzavery @markcowl I don't think this is truly a breaking change candidate since the new members being added to swagger have always been valid in the service (which was found because of a client break). This is a bug fix for missing enum values vs. a feature addition of new values in both the service and client.

Basically, just the client was missing these values as possibilities to be returned from the service. This is similar to other fixes we have made to make our swagger spec more accurately reflect the reality of the service as we move toward convergence between the client and service contract sharing the swagger spec (which has not yet happened, but we are getting closer every day)

Commit:
Bug fix for updating secrets in ADL

Missed this when fixing credentials. No objects are returned when
creating or updating secrets or credentials.

AutorestCI commented:
[![Build Status](https://travis-ci.org/AutorestCI/azure-sdk-for-python.svg?branch=RestAPI-PR844)](https://travis-ci.org/AutorestCI/azure-sdk-for-python) https://github.com/Azure/azure-sdk-for-python/pull/943

AutorestCI commented:
[![Build Status](https://travis-ci.org/AutorestCI/azure-sdk-for-node.svg?branch=RestAPI-PR844)](https://travis-ci.org/AutorestCI/azure-sdk-for-node) https://github.com/Azure/azure-sdk-for-node/pull/2020

AutorestCI commented:
No modification for Ruby
